### PR TITLE
Build merged Microsoft.Cci.dll against .NETStandard 1.3

### PR DESCRIPTION
**Pull request by nguerrera**
@mike-barnett @terrajobst @tijoytom @weshaggard 

I couldn't go as low as 1.1 or 1.2 as #2 requests, but this should help getting a new .nupkg built from here.

### CODE REVIEW

File: Microsoft.Cci.csproj [hunk 1]
```diff
@@ -41,16 +49,24 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|amd64' ">
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">
```
Comment (nguerrera) [inline Microsoft.Cci.csproj]: This needs to be != .NETStandard

Comment (nguerrera) [inline Microsoft.Cci.csproj]: Fixed

File: project.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿{
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
+    "System.Collections.NonGeneric": "4.0.1",
```
Comment (nguerrera) [inline project.json]: We should remove the non-generic collection usage, but wanted to start with minimal churn.

**Comment by tijoytom** [comment]
LGTM

**Comment by weshaggard** [comment]
LGTM

**Pull request merged**